src/audiomixerboard.cpp

```diff
@@ -436,6 +436,7 @@ void CChannelFader::Reset()
     plblCountryFlag->setVisible ( false );
     plblCountryFlag->setToolTip ( "" );
     cReceivedChanInfo = CChannelInfo();
+    cReceivedMIDIID   = INVALID_INDEX;
```

Comment:
Should we be using `INVALID_INDEX` here?

Reply:
Well it just needs to be something like `-1` that couldn't match a real MIDI Controller offset from `0` upwards. I don't mind whether we use that symbol or define another symbol for it, although I don't see the point of changing it.

Reply:
That's what I said.  I was told it was confusing so I changed it.  I'm raising here because it was one of the first objections on my pull request, so I'm assuming it was something important.